devices/stm32f401.yaml

```diff
@@ -12,6 +12,46 @@ _rebase:
       TIFRFE:
         name: "FRE"
 
+"I2C*":
+  _add:
+    # FLTR register defined RM0368
+    FLTR:
+      description: FLTR register
+      addressOffset: 0x24
+      access: read-write
+      resetValue: 0x0000
+      fields:
+        ANFOFF:
+          description: Analog noise filter
+          bitOffset: 4
+          bitWidth: 1
+        DNF:
+          description: Digital noise filter
+          bitOffset: 4
```

Comment:
```suggestion
          bitOffset: 0
```

Reply:
Pushed an update. Thank you for looking into all of this, very nice this applies to many other devices as well!

Sorry about the `bitOffset`, thanks for finding that error. I also noticed I had `ANFOFF` when the correct name is `ANOFF`. Fixed that.

Reply:
Thanks! And good spot on ANOFF. It turns out actually every other STM32 (those using the newer "v2" I2C peripheral) call it ANFOFF in their CR2 register, so maybe that's where you got it from? Some of the F4s that also include that newer I2C peripheral (generally I2C4 or FMP_I2C) also have ANFOFF. However, the reference manual is clear that all the F4s with the v1 peripheral and FLTR register call it ANOFF, so your patches seem good to me.

Reply:
Yea, I'm reading a [book](http://web.eece.maine.edu/~zhu/book/) that's using the STM32L4. Supposedly the L4 has the `ANFOFF` (as opposed to `ANOFF`) in CR1!